integration/auction_test.sh

```diff
@@ -21,37 +24,103 @@ CC_VERS=0
 num_rounds=3
 num_clients=10
 
+# Try function which returns the response string
+try_r() {
```

Comment:
if you return the result without filtering, this could potentially also be a useful function for common_utils.sh?

Reply:
Yes, will move try_r()  and check_result() to common_utils.sh